pkgs/development/ocaml-modules/alcotest/default.nix

```diff
@@ -1,18 +1,20 @@
 { lib, buildDunePackage, fetchurl
-, astring, cmdliner, fmt, uuidm, re, stdlib-shims, uutf
+, astring, cmdliner, fmt, uuidm, re, stdlib-shims, uutf, ocaml-syntax-shims
```

Comment:
This argument is unused.

Reply:
It's used on line 16

Reply:
Oh, sorry, you're right.  I was confused by GitHub's diff highlighting.  (More saturated background for words added on a line, but not for a completely-new line.  Weird.)

Reply:
No problem, it wasn't too unlikely that I would have missed this 😂